### DISCUSSION TITLE
http://www.icub.eu/ubuntu and http://www.icub.eu/debian deb repositories will be deactivated on 1st of January 2024

### DISCUSSION THREAD

**Discussion by traversaro** [robotology/community#653] [Announcements]
Hi @robotology/everyone !

As discussed in https://github.com/orgs/robotology/discussions/612 and announced in https://github.com/icub-tech-iit/documentation/issues/261, no new .deb binaries for `ycm-cmake-modules`, `yarp` and `icub-main` have been updated to the http://www.icub.eu/ubuntu and http://www.icub.eu/debian deb repositories since March 2023. However, until now the old packages uploaded before March 2023 are still available on http://www.icub.eu/ubuntu and http://www.icub.eu/debian .

The **http://www.icub.eu/ubuntu** and **http://www.icub.eu/debian** deb repositories will remain online until 1st of January 2024, date after which they are going to be deactivated. Please make sure to update any system or process of yours that depends on http://www.icub.eu/ubuntu or http://www.icub.eu/debian before that date, thanks!

**Comment by traversaro** [comment]
fyi @lornat75 @maggia80 @pattacini @Nicogene @mbrunettini

**Comment by pattacini** [comment]
cc @robotology/everyone

**Comment by mbrunettini** [comment]
**PLEASE NOTE** THAT the announcement above is valid for the DEBIAN repository [http://www.icub.eu/debian](http://www.icub.eu/debian) too.

**Reply by traversaro** [reply]
Thanks, I updated the original announcement to clarify that.

**Reply by mbrunettini** [reply]
Thanks!

**Comment by traversaro** [comment]
If due to this change you are experiencing errors like:
~~~
Err:1 http://www.icub.eu/ubuntu jammy InRelease
  Could not resolve 'www.icub.eu'
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: Failed to fetch http://www.icub.eu/ubuntu/dists/jammy/InRelease  Could not resolve 'www.icub.eu'
W: Some index files failed to download. They have been ignored, or old ones used instead.
~~~

You just need to delete the `www.icub.eu` repo from your sources. Either cancel the file `/etc/apt/sources.list.d/icub.list` :

~~~
sudo rm /etc/apt/sources.list.d/icub.list
~~~

or if that is not enough, remove any line containg `icub.eu` from `/etc/apt/sources.list` file.
